pom.xml

```diff
@@ -92,7 +92,7 @@
         <netty4.version>4.1.30.Final</netty4.version>
         <slf4j.version>1.7.12</slf4j.version>
         <!-- If compiling with different hadoop version also modify default hadoop coordinates in TaskConfig.java -->
-        <hadoop.compile.version>2.6.0</hadoop.compile.version>
+        <hadoop.compile.version>2.8.3</hadoop.compile.version>
```

Comment:
Reverting back to Default Version that ships with Druid.

Reply:
Do we need to update the version for `druid.indexer.task.defaultHadoopCoordinates` in `druid-private` MM runtime properties?